linumpy/io/test_data.py

```diff
@@ -23,34 +25,56 @@ def _create_linumpy_home_if_not_exists():
         os.mkdir(LINUMPY_HOME)
 
 
+def _get_mosaic_3d_nifti():
+    _create_linumpy_home_if_not_exists()
+    filename = os.path.join(LINUMPY_HOME, "mosaic_3d.nii")
+    if not os.path.exists(filename):
+        # create test data
+        data = np.mean(cells3d(), axis=1)  # (60, 256, 256)
+
+        img = nib.Nifti1Image(data)
```

Comment:
Missing an affine transform matrix. You can set the identity matrix:
`img = nib.Nifti1Image(data, np.eye(4))`